.github/workflows/concrete_compiler_publish_docker_images.yml

```diff
@@ -1,69 +1,62 @@
-# Build and publish Docker images for different applications using AWS EC2.
-name: Compiler - Docker images build & publish
+name: concrete-compiler publish docker images
 
 on:
   workflow_dispatch:
-    inputs:
-      instance_id:
-        description: 'Instance ID'
-        type: string
-      instance_image_id:
-        description: 'Instance AMI ID'
-        type: string
-      instance_type:
-        description: 'Instance product type'
-        type: string
-      runner_name:
-        description: 'Action runner name'
-        type: string
-      request_id:
-        description: 'Slab request ID'
-        type: string
-      matrix_item:
-        description: 'Build matrix item'
-        type: string
+  pull_request:
 
-# concurrency:
-#   group: compiler_publish_docker_images-${{ github.ref }}
-#   cancel-in-progress: true
+  push:
+    paths:
+      - .github/workflows/concrete_compiler_publish_docker_images.yml
+    branches:
+      - 'main'
+      - 'force-docker-images'
```

Comment:
As I mentioned in the main workflow, I don't think this keeps the same previous behavior. Mainly, we will be building docker images in PRs that don't change the image at all.

Reply:
right I will fix that :+1:

Reply:
done